templates/dns.yml.erb

```diff
@@ -17,7 +16,10 @@
 <%if scope.lookupvar("foreman_proxy::dns_provider") == 'nsupdate_gss' -%>
 :dns_tsig_keytab: <%= scope.lookupvar("foreman_proxy::dns_tsig_keytab") %>
 :dns_tsig_principal: <%= scope.lookupvar("foreman_proxy::dns_tsig_principal") %>
+# dns_key must be disabled if nsupdate_gss is used
+#:dns_key: <%= scope.lookupvar("foreman_proxy::keyfile") %>
```

Comment:
Nice :panda_face: 
Could you open a PR to theforeman/theforeman.org to fix chapter 4.3.6.3 then? :)